zenoh/src/net/routing/hat/linkstate_peer/network.rs

```diff
@@ -336,6 +337,7 @@ impl Network {
 
     pub(super) fn link_states(&mut self, link_states: Vec<LinkState>, src: ZenohId) -> Changes {
         log::trace!("{} Received from {} raw: {:?}", self.name, src, link_states);
+        let strong_runtime = self.runtime.upgrade().unwrap();
```

Comment:
Same question: why not to just immediately return empty `Changes` instead of panic?